R/cast.R

```diff
@@ -79,11 +79,16 @@
 #' # x is a double, but no information is lost
 #' vec_cast(1, integer())
 #'
-#' # Information is lost so a warning is generated
+#' # Information is lost in all conversions so an error is generated
 #' \dontrun{
 #' vec_cast(1.5, integer())
 #' }
 #'
+#' # Information is lost in some conversions so a warning is generated
+#' \dontrun{
```

Comment:
I think this can run, since it's only a warning?

Reply:
✅